### PR TITLE
fix: Improve escaping of literals and identifiers

### PR DESCRIPTION
**Pull request by TrevorBergeron**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: tests/system/small/test_remote_function.py [hunk 1]
```diff
@@ -724,7 +724,7 @@ def add_ints(row):
     )
 
 
-@pytest.mark.flaky(retries=2, delay=120)
+# @pytest.mark.flaky(retries=2, delay=120)
```
Comment (Genesis929) [inline tests/system/small/test_remote_function.py:727]: Why is this commented out?

**Review by Genesis929 — APPROVED** [review]

**Review by Genesis929 — APPROVED** [review]

File: bigframes/session/__init__.py [hunk 2]
```diff
@@ -1701,6 +1701,7 @@ def _start_query(
         Starts BigQuery query job and waits for results.
         """
         job_config = self._prepare_query_job_config(job_config)
+        print(sql)
```
Comment (Genesis929) [inline bigframes/session/__init__.py:1704]:
```suggestion
```

Comment (TrevorBergeron) [reply bigframes/session/__init__.py]: removed

File: tests/system/small/test_remote_function.py (hunk 1 above)
Comment (TrevorBergeron) [reply tests/system/small/test_remote_function.py]: was for testing on my machine. removed in new revision.

**Pull request merged**